lib/dialects/postgres/index.js

```diff
@@ -42,7 +42,7 @@ PostgresDialect.prototype.supports = _.merge(_.cloneDeep(AbstractDialect.prototy
     functionBased: true
   },
   inserts: {
-    onConflictDoNothing: ' ON CONFLICT DO NOTHING'
+    onConflictDoNothing: 'ON CONFLICT DO NOTHING'
```

Comment:
I don't believe the supports `EXCEPTION` is still in use with these changes. Maybe remove it entirely?

Reply:
Agree, exception handling requires temporary functions which are incompatible with binding parameters.